src/dotnet/Program.cs

```diff
@@ -125,6 +126,9 @@ internal static int ProcessArgs(string[] args, ITelemetry telemetryClient)
                 return 1;
             }
 
+            if (telemetryClient == null)
```

Comment:
nit: I don't know of any other places where we don't use brackets even for single line statements after `if`